src/main/java/org/jabref/logic/git/GitHandler.java

```diff
@@ -202,4 +220,116 @@ public String getCurrentlyCheckedOutBranch() throws IOException {
             return git.getRepository().getBranch();
         }
     }
+
+    /**
+     * Represents the Git status of a file within a repository
+     */
+    public enum GitStatus {
+        MODIFIED("Modified"),
+        STAGED("Staged"),
+        COMMITTED("Committed"),
+        UP_TO_DATE("Up to date"),
+        BEHIND_REMOTE("Behind remote"),
+        AHEAD_OF_REMOTE("Ahead of remote"),
+        UNTRACKED("Untracked");
```

Comment:
Code duplication (or maybe call it magic string duplication) with the Map in LibraryTab.
Is the displayName even needed anywhere or superfluous?